Doc/using/configure.rst

```diff
@@ -1012,7 +1012,7 @@ differently depending if the ``Py_BUILD_CORE_MODULE`` macro is defined:
 * Use ``Py_IMPORTED_SYMBOL`` otherwise.
 
 If the ``Py_BUILD_CORE_BUILTIN`` macro is used by mistake on a C extension
-built as a shared library, its ``PyInit_xxx()`` function is not exported,
```

Comment:
You can also use it for -I in: https://docs.python.org/dev/using/configure.html#envvar-CPPFLAGS

Compare with new doc: https://cpython-previews.readthedocs.io/en/latest/using/configure.html#cmdoption-arg-CPPFLAGS